src/client/dfuse/pil4dfs/hook.c

```diff
@@ -89,10 +89,16 @@ static uint64_t lib_base_addr[MAX_NUM_LIB];
 /* List of names of loaded libraries */
 static char   **lib_name_list;
 
+/* we set libc_version as DEFAULT_LIBC_VER when we fail to determine it from libc.so */
+#define DEFAULT_LIBC_VER 2.28
```

Comment:
this does not sound correct to me and it looks a bit hacky

Reply:
ok. I will check whether there exists an API to query libc version.

Reply:
Function gnu_get_libc_version() could be used to query glibc version.